diff -r 10b3a96ed952 CMakeLists.txt
--- a/CMakeLists.txt	Tue Sep 03 22:05:14 2013 +0200
+++ b/CMakeLists.txt	Thu Sep 05 13:00:25 2013 +0200
@@ -4,9 +4,9 @@
 
 # guard against in-source builds
 
-if(${CMAKE_SOURCE_DIR} STREQUAL ${CMAKE_BINARY_DIR})
-  message(FATAL_ERROR "In-source builds not allowed. Please make a new directory (called a build directory) and run CMake from there. You may need to remove CMakeCache.txt. ")
-endif()
+#if(${CMAKE_SOURCE_DIR} STREQUAL ${CMAKE_BINARY_DIR})
+#  message(FATAL_ERROR "In-source builds not allowed. Please make a new directory (called a build directory) and run CMake from there. You may need to remove CMakeCache.txt. ")
+#endif()
 
 # guard against bad build-type strings
 
