From 97c672fa5d04d2be270f3bca0e8f2e1c9b46f823 Mon Sep 17 00:00:00 2001
From: vmayoral <v.mayoralv@gmail.com>
Date: Sat, 27 Jul 2013 11:18:36 +0200
Subject: [PATCH] CMakeLists patch for cross-compiling

---
 rospy_tutorials/CMakeLists.txt |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/rospy_tutorials/CMakeLists.txt b/rospy_tutorials/CMakeLists.txt
index bcd4ddd..274eaf0 100644
--- a/rospy_tutorials/CMakeLists.txt
+++ b/rospy_tutorials/CMakeLists.txt
@@ -10,6 +10,7 @@ generate_messages(DEPENDENCIES std_msgs)
 
 catkin_package(CATKIN_DEPENDS message_runtime std_msgs)
 
+if(CATKIN_ENABLE_TESTING)
 foreach(T
     test/test-add-two-ints.launch
     test/test-peer-subscribe-notify.launch
@@ -20,6 +21,7 @@ foreach(T
     test/test-connection-header.launch)
   add_rostest(${T})
 endforeach()
+endif()
 
 # do not wildcard install files since the root folder of the package will contain a debian folder for releasing
 foreach(T
-- 
1.7.9.5

